woocommerce/woocommerce-gutenberg-products-block
# Create AddressForm component and change its fields dynamically based on country

Fixes #1730.

### Screenshots
_Checkout_
![Peek 2020-03-02 17-46](https://user-images.githubusercontent.com/3616980/75697752-d3290580-5cad-11ea-8166-3964525db38a.gif)

_Cart_
<img src="https://user-images.githubusercontent.com/3616980/75697939-15524700-5cae-11ea-803a-1413cadd28a3.png" alt="Screenshot" width="310" />

### How to test the changes in this Pull Request:

1. Create a page with the _Cart_ block and another one with the _Checkout_ block.
2. In the _Cart_ block frontend, click on `change address` so the address form opens.
3. In both pages, interact with the address form and try changing the country. Notice how the fields update to match the country options.
